### PR TITLE
Fix proposed enhancement #780

### PR DESCRIPTION
**Pull request by lwatta**
This will allow the user to specify the username that ssh will use to access the images
that are spawned.
The default username is 'ubuntu' but allow the user to override with --openstack_image_u
This change is specific for Openstack clouds
    - Ubuntu this is usually "ubuntu".
    - RHEL uses cloud-user

### CODE REVIEW

File: perfkitbenchmarker/providers/openstack/flags.py [hunk 1]
```diff
@@ -58,6 +58,9 @@
 flags.DEFINE_integer('openstack_volume_size', None,
                      'Size of the volume (GB)')
 
+flags.DEFINE_string('openstack_image_username', 'ubuntu',
+                    'Ssh username for cloud image. Defaults to ubuntu')
```
Comment (cmccoy) [inline perfkitbenchmarker/providers/openstack/flags.py]: The default will be printed by the help formatter. Could you remove that section?

**Comment by cmccoy** [comment]
Thanks! One minor comment.

**Comment by lwatta** [comment]
Just to be clear. You don't want the "Defaults to ubuntu" to be part of the help message?
Might be an hour or two before I can make that change. 

Sent from Yahoo Mail on Android 
 
  On Fri, Jan 8, 2016 at 7:35 AM, Connor McCoy<notifications@github.com> wrote:   
Thanks! One minor comment.

**Comment by cmccoy** [comment]
Exactly. Let me know if it's too much trouble - we can merge this and
update.

On Fri, Jan 8, 2016 at 7:41 AM, lwatta <notifications@github.com> wrote:

> Just to be clear. You don't want the "Defaults to ubuntu" to be part of
> the help message?
> Might be an hour or two before I can make that change.
>
> Sent from Yahoo Mail on Android
>
> On Fri, Jan 8, 2016 at 7:35 AM, Connor McCoy<notifications@github.com>
> wrote:
> Thanks! One minor comment.
>
>
> —
> Reply to this email directly or view it on GitHub.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/pull/788#issuecomment-170034516>
> .
>



-- 
Connor McCoy | Software Engineer | connormccoy@google.com | 206-876-1657

**Comment by lwatta** [comment]
No issue just won't have laptop access for a bit

Sent from Yahoo Mail on Android 
 
  On Fri, Jan 8, 2016 at 7:47 AM, Connor McCoy<notifications@github.com> wrote:   Exactly. Let me know if it's too much trouble - we can merge this and
update.

On Fri, Jan 8, 2016 at 7:41 AM, lwatta <notifications@github.com> wrote:

> Just to be clear. You don't want the "Defaults to ubuntu" to be part of
> the help message?
> Might be an hour or two before I can make that change.
>
> Sent from Yahoo Mail on Android
>
> On Fri, Jan 8, 2016 at 7:35 AM, Connor McCoy<notifications@github.com>
> wrote:
> Thanks! One minor comment.
>
>
> —
> Reply to this email directly or view it on GitHub.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/pull/788#issuecomment-170034516>
> .
>



-- 
Connor McCoy | Software Engineer | connormccoy@google.com | 206-876-1657

**Comment by skschneider** [comment]
@lwatta, I see you pushed an update to address @cmccoy's comment. We can respond faster if you also post a comment when you push an update (with default settings, this triggers an email notification).

The PR in its current state looks good to me, so I'll merge it in. Thank you for contributing!

**Pull request merged**